app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,10 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
```

Comment:
Not having ```require_tree .``` does break the CSS for active-admin.  I'll put it back and add a ticket for splitting it out later (possibly).